### PR TITLE
Resolves #2074

### LINKED ISSUE #2074: New script -- PRISM screen finder for MAXIS users
Hows abouts a script that helps MAXIS users navigate through PRISM?

@levesswe, care to takes? You can has.

### PR DESCRIPTION
**Pull request by levesswe**
This new script is intended to assist financial workers in navigating to popular screens in PRISM.  To use, simply press the button for the screen you are looking to go to when open in PRISM.  At the screen, press F1 in the case number, or MCI number field to search for and select the participant whose information you would like to see.

### CODE REVIEW

File: Script Files/NAV/OTHER NAV/NAV - PRISM SCREEN FINDER.vbs [hunk 1]
```diff
@@ -0,0 +1,71 @@
+'GATHERING STATS----------------------------------------------------------------------------------------------------
+name_of_script = "NAV - PRISM SCREEN FINDER.vbs"
+start_time = timer
+
+'LOADING ROUTINE FUNCTIONS FROM GITHUB REPOSITORY---------------------------------------------------------------------------
```
Comment (theVKC) [inline Script Files/NAV/OTHER NAV/NAV - PRISM SCREEN FINDER.vbs]: @levesswe: this is the PRISM FuncLib! We have a separate one for MAXIS scripts. You can see it on our README. :)

File: Script Files/NAV/OTHER NAV/NAV - PRISM SCREEN FINDER.vbs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  Text 10, 0, 250, 25, "Press a button below to navigate to PRISM screens.  Then press F1 in the case number or MCI number field to select the participant or case information you are looking for."
+EndDialog
+
+
+'THE SCRIPT----------------------------------------------------------------------------------------------------
+
+'Connect to BlueZone
+EMConnect ""
+
+CALL check_for_PRISM(FALSE)
```
Comment (theVKC) [inline Script Files/NAV/OTHER NAV/NAV - PRISM SCREEN FINDER.vbs]: Does this function also work in our FuncLib? If it doesn't, we should copy it in, @levesswe. :) That would be awesome and very appreciated.

**Comment by theVKC** [comment]
I left you some comments on the file, @levesswe, about the PRISM vs MAXIS FuncLib and function. I still like the idea of merging your FuncLib with ours, and switching to a unified block. I'd be willing to discuss modifying our block for that, because I want to remove my name and the word "alpha users" from the MAXIS scripts too.

In any case, please fix the FuncLib block reference to match MAXIS standards. Also, your pull request title is not following the MAXIS standards outlined here: https://github.com/MN-Script-Team/DHS-MAXIS-Scripts#anatomy-of-a-pull-request. I'll leave it up awaiting your correction.

Thanks Wendy!

**Comment by theVKC** [comment]
Also, we need an issue for this. Can you create a MAXIS issue for this so we can track it for release notes?

File: Script Files/NAV/OTHER NAV/NAV - PRISM SCREEN FINDER.vbs (hunk 2 above)
Comment (RobertFewins-Kalb) [inline Script Files/NAV/OTHER NAV/NAV - PRISM SCREEN FINDER.vbs]: Just did a check. It's in there. (We're githubbing in Dakota Co)

**Pull request merged**